saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -130,13 +135,19 @@ def save(cls, info, instance, cleaned_input):
                     'email_type': OrderEventsEmails.FULFILLMENT.value},
                 type=OrderEvents.EMAIL_SENT.value,
                 user=info.context.user)
+        return FulfillmentCreate(fulfillment=instance, order=order)
```

Comment:
I'm not sure this would work as the result of `save` method is not used anywhere in `ModelMutation`. To override the mutation response you could override `success_response`.